emerald-core/Cargo.toml

```diff
@@ -42,6 +42,7 @@ rust-crypto = "0.2"
 secp256k1 = "0.6"
 rand = "0.3"
 byteorder="1.0"
+ethabi = "2.0.0"
```

Comment:
good idea. i am going to close this PR and make a new one that matches the JSONRPC updates